### ISSUE TITLE
WebGL - Android can't acces to back camera

### ISSUE THREAD

**Issue by pablopalma** [EnoxSoftware/MarkerLessARExample#7]
Hello, 

I'm using the WebcamTextureMarklessARExample. When i run my WebGL scene through my Android device, i can't change my front camera to my back camera.

There's some common issue? Some solution?

Thanks.

**Comment by EnoxSoftware** [comment]
This issue may be related to the following issues.
https://forum.unity.com/threads/device-camera-in-webgl.523253/

**Comment by pablopalma** [comment]
Yes , i tried build to development build but i got this error that i think it's referred with OpenCV 
[error development build.txt](https://github.com/EnoxSoftware/MarkerLessARExample/files/3642259/error.development.build.txt)

**Comment by EnoxSoftware** [comment]
Currently, it seems that build error occurs when development build is enabled on Unity 2018.2 or higher WebGL platform.
It's under investigation.

**Comment by pablopalma** [comment]
I'm using 2019.1.x. So in 2018.1 or below should work the development build?

**Comment by EnoxSoftware** [comment]
Yes, if using Unity2018.1 or below, the development build will succeed without problems.

**Comment by pablopalma** [comment]
I'm having the following error: 

error - still waiting on run dependencies: UnityLoader.js:1091:40 dependency: wasm-instantiate

This error occurs when i compile with development build and i check de pre-built option. 

In the PC i can start my app, but in mobile crashes.

If i not check the pre-built option, the build fails with the emscripten py.

Some idea about this?

**Comment by EnoxSoftware** [comment]
Thank you very much for reporting.

Could you tell me about your test environment?
OpenCVForUnity version :
Unity version :

**Comment by pablopalma** [comment]
Unity 2018.1.9f2
OpenCV 2.3.6

**Comment by pablopalma** [comment]
Some news? i'm really complicated with deliver time, so if we can talk in some fludent way for quick responses, will great.

**Comment by EnoxSoftware** [comment]
Unity WebGL content is not currently supported on mobile devices.
https://docs.unity3d.com/Manual/webgl-browsercompatibility.html

The build succeeded without problems in my environment.
Unity 2018.1.9f2
OpenCV 2.3.6
development build on
use pre-built option on
https://www.dropbox.com/s/o98pk6679ms9jvz/OpenCVForUnityWebGL_Unity2018.1.9f2.zip?dl=0


Also,
Fixed the build error that occurred when development build was enabled on Unity 2018.2 or higher. 
https://www.dropbox.com/s/462xa2aayni7oeq/OpenCVForUnity_WebGLPlugin_DevelopmentBuild.unitypackage?dl=0
